src/actions/index.js

```diff
@@ -89,6 +90,11 @@ export const RESET_STATUS_TEXT = 'RESET_STATUS_TEXT';
 //
 // Action Creators
 //
+export const isOnlineCheck = (onlineBool: boolean) => ({
```

Comment:
I think it would be better to name this `onlineStatus` because I don't like the bool in the name and it's more readable.